File: server/java/src/main/java/com/onlinepayments/example/service/dtos/payments/CreatePaymentResponseDto.java
package com.onlinepayments.example.service.dtos.payments;

import com.onlinepayments.example.service.domain.common.enums.Status;
import com.onlinepayments.example.service.domain.common.StatusOutput;

public record CreatePaymentResponseDto(
        String id,
        Status status,
        StatusOutput statusOutput
) {}

File: server/dotnet/sdk-dotnet-example/Business/DTOs/CreatePayments/CreatePaymentResponseDto.cs
using Business.Domain.Enums.Payments;
using Business.Domain.Payments;

namespace Business.DTOs.CreatePayments;

public class CreatePaymentResponseDto
{
    public string? Id { get; init; }

    public Status? Status { get; init; }

    public StatusOutput? StatusOutput { get; init; } = new();
}